src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+       action="/search"
+       method="get"
+       data-qa="big"
+       class="form"
+       >
+      <label
+         for="big icon">
```

Comment:
The 'for' attribute of the label should match the 'id' of the input it is associated with. Avoid using spaces (' ') in 'id' or 'for' values. this should be just `big` or even better `input`